reference/configuration/framework.rst

```diff
@@ -515,7 +1169,7 @@ Whether or not to enable the ``translator`` service in the service container.
 fallback
 ........
 
-**default**: ``en``
+**type**: ``string`` **default**: ``'en'``
 
 This option is used when the translation key for the current locale wasn't found.
```

Comment:
Usually written english uses the long form. Either way, make it consistent through out the doc. At the moment you got both versions in there.